routes/signup.js

```diff
@@ -65,47 +68,65 @@ connection.connect(function(err){
     }
     });
   });
-  // router.post('/login',login.login)
-  // app.use('/api', router);
-
-// router.post('/SignUp', function(req, res) {
-//   // Get sent data.
-//   var user = req.body;
-//   // Do a MySQL query.
-//   connection.query('INSERT INTO users SET ?',
-//   { 
-//   username: user.userName,
-//   email: user.email,
-//   password: user.password,
-//   cyrptoProfil: user.cyrptoProfil,
-//   hasAgreed: user.hasAgreed
-//   },
-
-//   function(err, result) {
-//     // Neat!
-//   });
-//   res.end('Success');
-// });
-
-
-
 
-
-
-
-
-// router.post('/registered', function(req, res) {
-//   // Get sent data.
-//   // var user = req.body;
-//   // Do a MySQL query.
-//   connection.query('INSERT INTO users SET ?',req.body.username, req.body.email, req.body.password, 
-//   function(err, result) {
-//     if (error) throw error;
-//     res.json(result)
-    
-//   });
-//   res.redirect('/');
-// });
+// Encryption:
```

Comment:
I assume this file is not done yet. I'm seeing you're using mongodb syntax. @jace1084

Reply:
No, it’s not done, that’s why it’s commented out. I still need to add in the MySQL connection and take out the MongoDB